src/allmydata/scripts/runner.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import print_function
```

Comment:
Not fully ported, e.g. future imports aren't there.

Reply:
This is fine not to fully port, but if so it shouldn't be in `_python3` list of ported modules.

Reply:
Similar to when attempting to convert `scripts.common`, when doing a full port of `scripts.runner`, the conversion of bytes to unicode literals causes test failures at `test_create_alias:98`. So I guess I'll just punt on that too; shelving at https://github.com/tahoe-lafs/tahoe-lafs/tree/3603.scripts-runner-full and instead dropping runner from list of ported modules (6118d1a2d).